song-docs/source/introduction.rst

```diff
@@ -0,0 +1,156 @@
+==============
+Introduction
+==============
+
+
+What is SONG?
```

Comment:
Can you link back to Overture.bio somewhere ? (it doesn't have to be on that particular page, but should still be in a "visible" location)